examples/parse_rdf.py

```diff
@@ -59,6 +59,13 @@
             for lics in doc.extracted_licenses:
                 print '\tIdentifier: {0}'.format(lics.identifier)
                 print '\tName: {0}'.format(lics.full_name)
+            print 'Annotations:'
```

Comment:
for the examples to also work on Python 3, please import the future print function and use it.